test/test_transforms_v2.py

```diff
@@ -3398,6 +3398,23 @@ def test_transform(self, old_format, new_format, format_type):
             make_bounding_boxes(format=old_format),
         )
 
+    @pytest.mark.parametrize(("old_format", "new_format"), old_new_formats)
+    def test_strings(self, old_format, new_format):
```

Comment:
I opted to add a whole new test instead of re-purposing some of the existing ones, because in order for it to be a proper non-regression test, we need to validate the output values here. In `main`, we *can* pass strings, there's no failure: but the results are wrong, and the current existing tests cannot assert that properly.